docs/readme.md

```diff
@@ -4,38 +4,46 @@ displayed_sidebar: docs
+### Why Airbyte?
+ 
+Today, teams and organizations require efficient and timely data access to an ever-growing list of data sources. In-house data pipelines are brittle and costly to build and maintain. Airbyte's unique open-source approach enables your data stack to adapt as your data needs evolve.
+
+- **Wide connector availability:** Airbyte’s connector catalog comes “out-of-the-box” with over 350 pre-built connectors. These connectors can be used to start replicating data from a source to a destination in just a few minutes. 
+- **Long-tail connector coverage:** You can easily extend Airbyte’s functionality to support your custom use cases through Airbyte's [No-Code Connector Builder](./connector-development/connector-builder-ui/overview).
+
+- **Robust platform** provides horizontal scaling required for large-scale data movement operations, available as [Cloud-managed](https://airbyte.com/product/airbyte-cloud) or [Self-managed](https://airbyte.com/product/airbyte-enterprise).
+
+- **Accessible User Interfaces** through the UI, [**PyAirbyte**](./using-airbyte/pyairbyte/getting-started) (Python library), [**API**](./api-documentation), and [**Terraform Provider**](./terraform-documentation) to integrate with your preferred tooling and approach to infrastructure management.
 
 Airbyte is suitable for a wide range of data integration use cases, including AI data infrastructure and EL(T) workloads. Airbyte is also [embeddable](https://airbyte.com/product/powered-by-airbyte) within your own application or platform to power your product.
 
-## Airbyte Demo: Creating Connections
+### Airbyte Demo: Creating Connections
 
-<Arcade id="0k75Pa9c9EvrJb8zFsuU" title="Airbyte Demo: Get Started Creating Connections" paddingBottom="calc(61.416666666666664% + 41px)" />
+<Arcade id="8UUaeQOILatZ38Rjh8cs" title="Airbyte Demo: Get Started Creating Connections" paddingBottom="calc(61.416666666666664% + 41px)" />
 
-## For Airbyte Cloud users
+## Get Started
 
-Browse the [connector catalog](./integrations/) to find the connector you want. In case the connector is not yet supported on Airbyte Cloud, consider using [Airbyte Open Source](#for-airbyte-open-source-users).
+<Grid columns="3">
 
-Next, check out the [step-by-step tutorial](./using-airbyte/getting-started) to sign up for Airbyte Cloud, understand Airbyte [concepts](./using-airbyte/core-concepts), and run your first sync.
+<CardWithIcon title="Start Syncing Data" description="Deploy locally or sign up for Airbyte Cloud to sync data in minutes" ctaText="Get Started" ctaLink="./using-airbyte/getting-started/" icon="enterprise" />
 
-## For Airbyte Open Source users
+<CardWithIcon title="Connector Catalog" description="Browse the extensive Connector Catalog of over 350+ sources and destinations" ctaText="Browse" ctaLink="./integrations/" icon="cloud" variant="secondary" />
```

Comment:
@nataliekwong I changed the property name `variant` to `ctaVariant`.

Reply:
That worked, thank you!